site/source/docs/compiling/Dynamic-Linking.rst

```diff
@@ -185,6 +168,22 @@ when doing load time dynamic linking.
 
 There is also the corresponding ``-sSIDE_MODULE=2`` for side modules.
 
+System Libraries
+================
+
+As mentioned earlier, system libraries are handled in a special way by the
+Emscripten linker, and in dynamic linking, only the main module is linked
+against system libraries. With ``-sMAIN_MODULE=2`` side module dependencies are
+automatically handled, but with ``-sMAIN_MODULE=1`` it is possible that a side
```

Comment:
Maybe elaborate that they are automatically handled because the user must pass them in on the commandline when building the main module? I had to reread the docs above to realize that.